pkg/nemesis/time_chaos.go

```diff
@@ -0,0 +1,96 @@
+package nemesis
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+type TimeChaosLevels = int
+
+const (
+	SmallSkews TimeChaosLevels = iota
+	SubCriticalSkews
+	CriticalSkews
+	BigSkews
+	HugeSkews
+	// Note: StrobeSkews currently should be at end of iota system,
+	//  because TimeChaosLevels will be used as slice index.
+	StrobeSkews
+)
+
+const StrobeSkewsBios = 200
+
+type ChaosDurationType int
+
+const (
+	FromZero ChaosDurationType = iota
+	FromLast
+)
+
+const MsToNS uint = 1000000
+const SecToNS uint = 1e9
+const SecToMS uint = SecToNS / MsToNS
+
+// skewTimeMap stores the
+var skewTimeMap []uint
+var skewTimeStrMap map[string]TimeChaosLevels
+
+func init() {
+	skewTimeMap = make([]uint, 0)
+	skewTimeMap = append(skewTimeMap, 0)
+	skewTimeMap = append(skewTimeMap, 100)
+	skewTimeMap = append(skewTimeMap, 200)
+	skewTimeMap = append(skewTimeMap, 250)
+	skewTimeMap = append(skewTimeMap, 500)
+	skewTimeMap = append(skewTimeMap, 5000)
```

Comment:
Although this is ok, but is it better to directly use `skewTimeMap = []uint{0, 100,...}` ?

Reply:
Done, PTAL